pkg/scheduler/internal/queue/scheduling_queue.go

```diff
@@ -344,9 +344,13 @@ func (p *PriorityQueue) flushBackoffQCompleted() {
 		boTime, found := p.podBackoff.GetBackoffTime(nsNameForPod(pod))
 		if !found {
 			klog.Errorf("Unable to find backoff value for pod %v in backoffQ", nsNameForPod(pod))
-			p.podBackoffQ.Pop()
-			p.activeQ.Add(rawPodInfo)
-			defer p.cond.Broadcast()
+			_, err := p.podBackoffQ.Pop()
```

Comment:
IMO this fix is unnecessary. We've held the lock in L335, and in L339 we "peeked" the top element from the internal heap, then that means `p.podBackoffQ.Pop()` will definitely pop the "peeked" obj out, without any error.

Reply:
Please take a look at similar operation on line 356.

At least we should keep the handling consistent.

Reply:
@Huang-Wei 
Please provide your opinion.

Thanks